### PR TITLE
German label - Use funding eligibility instead of locale

### PR DESCRIPTION
**Pull request by alan-bendjuya**
Locale works when is set in the SDK params but the label isn’t dependent on it.

### CODE REVIEW

**Comment by alan-bendjuya** [comment]
Continuation of this [PR](https://github.com/paypal/paypal-checkout-components/pull/1687)

**Review by elizabethmv — COMMENTED** [review]
I pulled it down locally and looks good, thanks for fixing! 
Screen Shot 2021-07-13 at 2 14 23 PM

one note, should we add/modify the screenshot test config to have one for 300px?

**Comment by gregjopa** [comment]
> one note, should we add/modify the screenshot test config to have one for 300px?

Great suggestion @elizabethmv! This would be a great screenshot test to add.

File: test/screenshot/config.js [hunk 1]
```diff
@@ -914,3 +914,80 @@ buttonConfigs.push({
...
+            eligible: true,
+            products: {
+                paylater: {
+                    eligible: true,
+                    variant:  'DE'
+                }
+            }
+        }
+    }
+});
```
Comment (gregjopa) [inline test/screenshot/config.js:993]: Nit: you need a blank line at the end of this file

**Review by elizabethmv — APPROVED** [review]

**Review by gregjopa — APPROVED** [review]

**Pull request merged**